packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -30,12 +30,12 @@ IFS=, read -r -a origRpcAddrs <<<"$(AG_SETUP_COSMOS_HOME=$thisdir ag-setup-cosmo
 
 rpcAddrs=(${origRpcAddrs[@]})
 while [[ ${#rpcAddrs[@]} -gt 0 ]]; do
-  r=$(( $RANDOM % ${#rpcAddrs[@]} ))
+  r=$(( RANDOM % ${#rpcAddrs[@]} ))
```

Comment:
TIL: `echo $(( RANDOM ))` is a bash-ism
TIDNWTLMB. (_Today I Did Not Want To Learn More Bash-isms_:)

Reply:
Shellcheck made me do it.